piptools/resolver.py

```diff
@@ -429,9 +429,9 @@ def get_best_match(self, ireq: InstallRequirement) -> InstallRequirement:
 
         # Format the best match
         log.debug(
```

Comment:
Logging is supposed to pass params through dedicated arguments, though. Instead of pre-rendering strings before passing them into logging..